src/pages/dev/axelarjs-sdk/tx-status-query-recovery.mdx

```diff
@@ -137,13 +161,13 @@ Possible response values are:
 }
 ```
 
-#### 2. Increase gas payment
+## Increase gas payment
 
-There are two different functions to increase gas payment depending on type of the token.
+You can increase the gas payment using a native token or an ERC-20 token.
 
-##### 2.1 Native gas payment
+### Native gas payment
```

Comment:
Is this subheading necessary still? Only way to increase payment is native gas payment i dont think we support non-native gas payment anymore